css/_base.scss

```diff
@@ -51,6 +51,22 @@
             ol {
               margin-left: 20px;
             }
+
+            a {
+              color:teal;
+            }
+
+            a:hover {
+              color: lighten(teal, 15%);
+              -o-transition:.2s;
```

Comment:
Usually, you would specify the transition property, so in this case it should be transition: color 0.2s